tests/backend_test.py

```diff
@@ -505,9 +531,10 @@ def test_shots_bits_edgecases(n_shots, n_bits) -> None:
     assert res.get_counts() == correct_counts
 
 
+@pytest.mark.skip("Will be moved to a set of long running integration tests")
```

Comment:
Is there already an issue to capture this task?

Reply:
There is one now :) #147